### PR TITLE
docs: translate ko/spec/ics-024-host-requirements/README.md

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

**Comment by junbeomlee** [comment]
@mossid !!

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+
+### 지향 속성
+
+IBC는 정확히 구현하기 위해 상태 머신에게 가능한 간단한 인터페이스를 요구해야 합니다.
+
+## 기술적 명세
+
+### 모듈 시스템
+
+호스트 상태 머신은 반드시 모듈 시스템을 지원해야 합니다. 이를 통해 상호 신뢰할 수 없는 코드 패키지가 동일한 원장에서 독립적이고, 잠재적으로 안전하게 실행될 수 있습니다. 또한 다른 모듈과 통신 할 수 있는 방법과 시기를 제어할 수 있고, "마스터 모듈" 또는 실행 환경에 의해 식별 및 조작될 수 있습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
호스트 상태 머신은 반드시 모듈 시스템을 지원해야 합니다. 이를 통해 잠재적으로 상호 신뢰할 수 없는 독립적인 코드 패키지가 동일한 원장에서 안전하게 실행될 수 있습니다. 또한 다른 모듈과 통신 할 수 있는 방법과 시기를 제어할 수 있고, "마스터 모듈" 또는 실행 환경에 의해 식별 및 조작될 수 있습니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+IBC/TAO 사양은 핵심적인 "IBC handler" 모듈과 "IBC relayer" 모듈의 구현을 정의합니다. IBC/APP 사양은 어플리케이션 로직을 처리하는 특정 패킷을 위한 다른 모듈을 추가로 정의합니다. IBC는 "마스터 모듈" 또는 실행 환경을 사용하여 호스트 상태 머신의 다른 모듈에 IBC 핸들러 모듈 또는 IBC 라우팅 모듈에 대한 접근 권한을 부여할 수 있습니다. 부여하지 않는다면 상태 머신에 공존할 수 있는 다른 모듈의 기능 또는 통신 기능에 대한 요구사항을 부과하지 않습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
IBC/TAO 사양은 핵심적인 "IBC handler" 모듈과 "IBC relayer" 모듈의 구현을 정의합니다. IBC/APP 사양은 특정 패킷을 처리하는 어플리케이션 로직을 위한 다른 모듈을 추가로 정의합니다. IBC는 "마스터 모듈" 또는 실행 환경이 호스트 상태 머신의 다른 모듈에 IBC 핸들러 모듈 또는 IBC 라우팅 모듈에 대한 접근 권한을 부여할 수 있기를 요구하지만, 그 외에는 상태 머신에 공존할 수 있는 다른 모듈의 기능 또는 통신 기능에 대한 요구사항을 부과하지 않습니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+### Paths, identifiers, separators
+
+`식별자` 는 연결, 채널 또는 라이트 클라이언트와 같이 상태에 저장된 객체의 키로 사용되는 바이트 문자열입니다. 식별자는 영숫자로만 구성되어야 합니다. Identifier 는 (양의 정수 길이로) 비어있지 않아야 합니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
`식별자` 는 연결, 채널 또는 라이트 클라이언트와 같이 상태에 저장된 객체의 키로 사용되는 바이트 문자열입니다. 식별자는 영숫자로만 구성되어야 합니다. 식별자는 (양의 정수 길이로) 비어있지 않아야 합니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+`경로` 는 상태에 저장된 객체의 키로 사용되는 바이트 문자열입니다. 경로는 식별자, 상수 영숫자 문자열 및 구분자 `"/"` 만 포함해야 합니다.
+
+식별자는 가치있는 자원으로 의도된 것이 아니며 name squatting을 방지하기위해, 최소 길이 요구 사항 또는 의사 난수 생성이 구현 될 수 있지만 해당 사양에 대해 특정 제한이 적용되지 않습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
식별자는 가치있는 자원으로 의도된 것이 아니며 이름 점유 경쟁을 방지하기 위해, 최소 길이 요구 사항 또는 의사 난수 생성이 구현 될 수 있지만, 이 사양은 그러한 제약을 정의하지는 않습니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+
+`"/"` 구분자는 두 개의 식별자 또는 식별자와 상수 바이트 문자열을 분리하고 연결하는 데 사용됩니다. 식별자는 모호성을 방지하는 `"/"` 문자를 포함해서는 안됩니다.
+
+중괄호로 표시되는 변수 보간(variable interpolation)은 이 사양 전체에서 경로 형식을 정의하는 약칭으로 사용되며, 예시는`client/{clientIdentifier}/consensusState` 다음과 같습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
중괄호로 표시되는 변수 보간(variable interpolation)은 이 사양 전체에서 경로 형식을 정의하는 약칭으로 사용되며, 예시는`client/{clientIdentifier}/consensusState`와 같습니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+```typescript
+type delete = (path: Path) => void
+```
+
+`Path`는 위에서 정의한 대로 입니다. `Value`는 특정 데이터 구조를 인코딩한 임의의 바이트 문자열 입니다. 인코딩 세부 사항은 다른 ICS에서 다룹니다.
+
+이러한 함수는 IBC 핸들러 모듈 (구현은 별도의 표준으로 설명 됨)에만 권한이 있어야 하므로, IBC 핸들러 모듈만 (`get`을 통해  조회될 수 있는) path를 `set` 하거나 `delete`할 수 있습니다. 이것은 전체 상태 머신이 사용하는 더 큰 키/값 저장소의 (접두사 key를 가지는) 하위 저장소로 구현 될 수 있습니다.
+
+호스트 상태 머신은 반드시 이 인터페이스의 두 인스턴스를 제공해야합니다.
+그것들은 다른 체인이 읽을 (즉, 입증 된) 스토리지를 위한 `provableStore` 와 `get`, `set` 및   `delete`를 호출 할 수 있는 호스트에 로컬로 저장하기위한 `privateStore `이며, 예는 `provableStore.set('some/path', 'value')` 와 같습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
다른 체인이 읽을 (즉, 검증 된) 스토리지를 위한 `provableStore`, 그리고 호스트 자신의 스토리지이며 `get`, `set` 및 `delete`를 호출 할 수 있는(예: `provableStore.set('some/path', 'value') `privateStore`입니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+
+The `provableStore`:
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
`provableStore`는:
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+
+- [ICS 23](../ics-023-vector-commitments)에 정의된 vector commitment로 외부에서 입증 할 수 있는 데이터를 key/value store에 기록해야합니다.
+- 이 사양에 제공된 표준 데이터 구조 인코딩을 proto3로 사용해야합니다.
+
+The `privateStore`:
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
`privateStore`는:
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+`Path`는 위에서 정의한 대로 입니다. `Value`는 특정 데이터 구조를 인코딩한 임의의 바이트 문자열 입니다. 인코딩 세부 사항은 다른 ICS에서 다룹니다.
+
+이러한 함수는 IBC 핸들러 모듈 (구현은 별도의 표준으로 설명 됨)에만 권한이 있어야 하므로, IBC 핸들러 모듈만 (`get`을 통해  조회될 수 있는) path를 `set` 하거나 `delete`할 수 있습니다. 이것은 전체 상태 머신이 사용하는 더 큰 키/값 저장소의 (접두사 key를 가지는) 하위 저장소로 구현 될 수 있습니다.
+
+호스트 상태 머신은 반드시 이 인터페이스의 두 인스턴스를 제공해야합니다.
+그것들은 다른 체인이 읽을 (즉, 입증 된) 스토리지를 위한 `provableStore` 와 `get`, `set` 및   `delete`를 호출 할 수 있는 호스트에 로컬로 저장하기위한 `privateStore `이며, 예는 `provableStore.set('some/path', 'value')` 와 같습니다.
+
+The `provableStore`:
+
+- [ICS 23](../ics-023-vector-commitments)에 정의된 vector commitment로 외부에서 입증 할 수 있는 데이터를 key/value store에 기록해야합니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
- [ICS 23](../ics-023-vector-commitments)에 정의된 vector commitment를 사용하여 외부에서 검증할 수 있는 key/value store에 데이터를 기록해야합니다.
```

File: translation/ko/spec/ics-024-host-requirements/README.md [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+- 이 사양에 제공된 표준 데이터 구조 인코딩을 proto3로 사용해야합니다.
+
+The `privateStore`:
+
+- 외부 증명을 지원할 수도 있지만 반드시 필요한 것은 아닙니다. IBC 핸들러는 증명해야 할 데이터를 절대로 기록하지 않습니다.
```
Comment (mossid) [inline translation/ko/spec/ics-024-host-requirements/README.md]:
```suggestion
- 외부 증명을 지원할 수도 있지만 반드시 필요한 것은 아닙니다. IBC 핸들러는 검증이 필요한 데이터를 절대로 이 저장소에 기록하지 않습니다.
```

**Comment by cwgoes** [comment]
Bump @junbeomlee @mossid

**Comment by cwgoes** [comment]
Do you still intend to continue this? Otherwise I'll close this PR.

**Pull request closed without merging**